docs/_blog/liana-payjoin.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Automatically Refresh Inheritance Time-Locks with Payjoin"
```

Comment:
I've seen "Timelock" [everywhere](https://en.bitcoin.it/wiki/Timelock) and never Time-lock. Is that standard / search optimized?

Reply:
either that or [with space](https://bitcoinops.org/en/topics/htlc/)

Reply:
more [timelock](https://blog.casa.io/bitcoin-multisig-time-locking-challenges/)

Reply:
found a [Time-locked](https://thebitcoinmanual.com/btc-finance/tlc/) instance, but I'll let you decide how this matches in authority compared to the prior comments

Reply:
I'd prefer "timelock" no space or hyphen.

Reply:
And similarly "time-locked" -> "time locked" as two words